test/NuGet.Core.Tests/NuGet.Packaging.Test/NupkgMetadataFileFormatTests.cs

```diff
@@ -2,8 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.IO;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
+using System.Text;
```

Comment:
I'm wondering if we need to add a test that ensures the reading/writing among the 2 different implementations hasn't regressed?

Reply:
There's already one test that validates round tripping from write back to read, and another test that validates against a hard-coded json string. I think that's sufficient.

Also, our product code doesn't actually use the Newtonsoft.Json overloads. There's basically no reason for anyone else to read/write the `.nupkg.metadata` file (so I'm a little disappointed nobody encouraged me to [delete the Newtonsoft.Json overloads](https://github.com/NuGet/NuGet.Client/pull/6083#discussion_r1788386243)).

So, IMO, there isn't really a benefit, the NJ code is effectively dead code, if it wasn't a public API that someone could theoretically use.